Regressions in test_boolean.py with Blender 4.2.0

The following regressions appear to be associated with the update from Blender 4.1.1 to 4.2.0 in Fedora 40, the current latest stable release. (We didn’t first see these regressions in the upcoming Fedora 41 or in Rawhide first because the maintainers of the `blender` package in Fedora have not yet been able to build it with Python 3.13.)

```
=================================== FAILURES ===================================
___________________________ BooleanTest.test_boolean ___________________________
[gw2] linux -- Python 3.12.4 /usr/bin/python3   

self = <tests.test_boolean.BooleanTest testMethod=test_boolean>

    def test_boolean(self):
        a, b = self.a, self.b

        times = {}
        for engine, exists in engines:
            # if we have all_dep set it means we should fail if
            # engine is not installed so don't continue
            if not exists:
                g.log.warning("skipping boolean engine %s", engine)
                continue

            g.log.info("Testing boolean ops with engine %s", engine)

            tic = g.time.time()

            # do all booleans before checks so we can time the backends
            ab = a.difference(b, engine=engine)
            ba = b.difference(a, engine=engine)
            i = a.intersection(b, engine=engine)
            u = a.union(b, engine=engine)

            times[engine] = g.time.time() - tic

>           assert ab.is_volume
E           assert False
E            +  where False = <trimesh.Trimesh(vertices.shape=(0, 3), faces.shape=(0, 3))>.is_volume

tests/test_boolean.py:49: AssertionError
__________________________ BooleanTest.test_multiple ___________________________
[gw2] linux -- Python 3.12.4 /usr/bin/python3

self = <tests.test_boolean.BooleanTest testMethod=test_multiple>

    def test_multiple(self):
        """
        Make sure boolean operations work on multiple meshes.
        """
        for engine, exists in engines:
            if not exists:
                continue
            a = g.trimesh.primitives.Sphere(center=[0, 0, 0])
            b = g.trimesh.primitives.Sphere(center=[0, 0, 0.75])
            c = g.trimesh.primitives.Sphere(center=[0, 0, 1.5])

            r = g.trimesh.boolean.union([a, b, c], engine=engine)

>           assert r.is_volume
E           assert False
E            +  where False = <trimesh.Trimesh(vertices.shape=(0, 3), faces.shape=(0, 3))>.is_volume

tests/test_boolean.py:82: AssertionError
```

…

```
=========================== short test summary info ============================
FAILED tests/test_boolean.py::BooleanTest::test_boolean - assert False
FAILED tests/test_boolean.py::BooleanTest::test_multiple - assert False
============ 2 failed, 623 passed, 19 warnings in 248.03s (0:04:08) ============
```

I don’t really have any further insight into this, but I’m happy to provide any other information that might be helpful.

--- a/trimesh/resources/templates/blender_boolean.py.tmpl
+++ b/trimesh/resources/templates/blender_boolean.py.tmpl
@@ -40,7 +40,7 @@ if __name__ == '__main__':
     mesh_post = os.path.abspath(r'$MESH_POST')
 
     for filename in mesh_pre:  # use data.objects instead of context.scene.objects
-        bpy.ops.import_mesh.stl(filepath=os.path.abspath(filename))
+        bpy.ops.wm.stl_import(filepath=os.path.abspath(filename))
 
     mesh = bpy.data.objects[0]
     # Make sure mesh is the active object
@@ -62,6 +62,6 @@ if __name__ == '__main__':
         bpy.ops.object.modifier_apply(modifier=mod.name)
 
     delete_nonresult(bpy)
-    bpy.ops.export_mesh.stl(
+    bpy.ops.wm.stl_export(
         filepath=mesh_post,
-        use_mesh_modifiers=True)
+        apply_modifiers=True)

